Foundation/HTTPCookieStorage.swift

```diff
@@ -183,7 +194,7 @@ open class HTTPCookieStorage: NSObject {
     }
     
     open override var description: String {
-        return "<NSHTTPCookieStorage cookies count:\(cookies?.count ?? 0)>"
+        return isEphemeral ? "Ephemeral" : "" + "<NSHTTPCookieStorage cookies count:\(cookies?.count ?? 0)>"
```

Comment:
string concatenation is expensive and most of the times lesser performant to string interpolation. you could use string interpolation instead.

`"\(isEphemeral ? "Ephemeral" : "")<NSHTTPCookieStorage cookies count:\(cookies?.count ?? 0)>"`

Reply:
@manojmahapatra Done. Thanks!